docs/antora-container-playbook-dev.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Note: this playbook is only used for OSS builds. Non-OSS builds use antora-container-playbook-proprietary instead.
+site:
+    title: Couchbase Monitoring and Observability Stack
+    url: /cmos
+    start_page: cmos::index.adoc
+content:
+    sources:
+        - url: ./
+          branches: HEAD
+          start_path: docs
+runtime:
+    cache_dir: ./build/cache
+    log:
+      level: all
+ui:
+    bundle:
+        url: https://gitlab.com/antora/antora-ui-default/-/jobs/artifacts/master/raw/build/ui-bundle.zip?job=bundle-stable
+asciidoc:
+    attributes:
+        env-packaged: ''
+        env-oss: ''
```

Comment:
Do we want `env-oss` to be enabled at dev-time? It'll strip out all the non-OSS docs.

Reply:
So currently `make docs` is built off of the OSS playbook so it's OSS docs only. This causes various Antora warnings because of references to non-OSS attributes like `cluster_monitor_user` and CBMM pages.
I'll be moving this target to `make docs-oss` and `make docs` as non-OSS in time, just requires copying in cbmultimanager repo etc.

Reply:
It's fine to leave it as OSS (at least until we work out what we're doing with the CBMM build) - perhaps worth a note in contributing.md?